### PR TITLE
Taur and borgo feature shieet!

### PR DESCRIPTION
**Pull request by Verkister**
-Added a counter for janihound to keep track of how much crap it's ate.
-Lowered janigut capacity to 25 items.
-Made mice edible for the borgo.
-Gave a gentle touch to the wolftaur sprites. (more types later?)
-Added a saddlebag version that fits itself for every quadtaur.
-Added a robust combat aesthetic saddlebag as a childtype to the forementioned.
-Also attempted to make it so that people can feed crap for the borgo. Unable to test locally for obvious reasons (2 for tango etc.) so no guarantees on that mechanic.
-Added proper icons for north facing loafers.
-Slightly reduced the south facing fed loafers' gut spread.
-Fixed direction issue in fed default sprite.
-Added a new type for Ace's outdoors plans. (icon state "siftusian")

### CODE REVIEW

File: code/modules/mob/living/simple_animal/vore/otie.dm [hunk 1]
```diff
@@ -26,22 +26,32 @@
...
+	friendly = list("nuzzles", "slobberlicks", "noses softly at", "noseboops", "headbumps against", "leans on", "nibbles affectionately on")
+	meat_amount = 6
 	old_x = -16
 	old_y = 0
 	pixel_x = -16
 	pixel_y = 0
 
 	var/mob/living/carbon/human/friend
 	var/tamed = 0
+	var/list/friendmobs = list(
```
Comment (SpadesNeil) [inline code/modules/mob/living/simple_animal/vore/otie.dm]: There's a much more effective way to handle this, which is currently being resolved in https://github.com/VOREStation/VOREStation/commit/4367bc2da762a7acf5316e7cfcfadf6be5172485

Can you discard these changes?

File: code/modules/mob/living/simple_animal/vore/otie.dm [hunk 2]
```diff
@@ -93,6 +103,8 @@
 			return null
 		else if(tamed == 1 && isrobot(found_atom))
 			return null
+		else if(tamed == 1 && is_type_in_list(found_atom, friendmobs))
```
Comment (SpadesNeil) [inline code/modules/mob/living/simple_animal/vore/otie.dm]: Also discard these.

File: code/modules/mob/living/simple_animal/vore/otie.dm [hunk 3]
```diff
@@ -114,6 +126,8 @@
...
```
Comment (SpadesNeil) [inline code/modules/mob/living/simple_animal/vore/otie.dm]: And these.

File: code/modules/mob/living/simple_animal/vore/otie.dm (hunk 1 above)
Comment (Verkister) [inline code/modules/mob/living/simple_animal/vore/otie.dm]: Alright.

File: code/modules/mob/living/simple_animal/vore/otie.dm [hunk 4]
```diff
@@ -26,33 +26,22 @@
...
-	say_got_target = list("Rurrr!", "ROAR!", "RERR!", "NOM!", "MINE!", "RAHH!", "RAH!", "WARF!")
+	say_got_target = list("Rurrr!", "ROAR!", "MINE!", "RAHH!", "Slurp.. RAH!")
 	melee_damage_lower = 5
 	melee_damage_upper = 20
 	response_help = "pets the"
 	response_disarm = "bops the"
 	response_harm = "hits the"
 	attacktext = "mauled"
-	friendly = list("nuzzles", "slobberlicks", "noses softly at", "noseboops", "headbumps against", "leans on", "nibbles affectionately on")
-	meat_amount = 6
```
Comment (SpadesNeil) [inline code/modules/mob/living/simple_animal/vore/otie.dm]: You didn't have to revert these lines ya doof.

File: code/modules/mob/living/simple_animal/vore/otie.dm [hunk 5]
```diff
@@ -26,33 +26,22 @@
 	emote_hear = list("rurrs", "rumbles", "rowls", "groans softly", "murrs", "sounds hungry", "yawns")
 	emote_see = list("stares ferociously", "snarls", "licks their chops", "stretches", "yawns")
 	say_maybe_target = list("Ruh?", "Waf?")
-	say_got_target = list("Rurrr!", "ROAR!", "RERR!", "NOM!", "MINE!", "RAHH!", "RAH!", "WARF!")
```
Comment (SpadesNeil) [inline code/modules/mob/living/simple_animal/vore/otie.dm]: Or these.

**Comment by SpadesNeil** [comment]
calling on @Arokha to review stuff since I did a terrible job last time

**Pull request merged**